examples/embedded_devices/base_images/gpu/Dockerfile

```diff
@@ -2,25 +2,12 @@
 # We also install dependencies for a very exciting future project
 
 # check this repo to learn more about the image below: https://github.com/dusty-nv/jetson-containers
-FROM nvcr.io/nvidia/l4t-pytorch:r32.4.4-pth1.6-py3
+FROM nvcr.io/nvidia/l4t-pytorch:r32.5.0-pth1.6-py3
 
 RUN mkdir /app
 WORKDIR /app
 
 # update pip
-RUN python3 -m pip install --upgrade pip
-
-# Installing flower
-RUN pip3 install flwr==0.13.0
```

Comment:
Is `flwr` installed in a later step instead anyway?

Reply:
makes sense 👍